templates/resources/vmaas_load_balancer.md.tmpl

```diff
@@ -0,0 +1,23 @@
+---
+layout: ""
+page_title: "hpegl_vmaas_load_balancer Resource - vmaas-terraform-resources"
+subcategory: {{ $arr := split .Name "_" }}"{{ index $arr 1 }}"
+description: |-
+  {{ .Description | plainmarkdown | trimspace | prefixlines "  " }}
+---
+
+# Resource hpegl_vmaas_load_balancer
+
+-> Compatible version >= 5.4.6
+
+{{ .Description | trimspace }}
+`hpegl_vmaas_load_balancer` resource supports NSX-T Load Balancer creation.
+
+-> It is mandatory to choose the `Tier1 Gateway` while creating the load balancer
```

Comment:
why do you need talk about Tier1 ? does the schema doesnt say its a required field ?

Reply:
here it is not required but for creating the virtual server from LB without Tier1 will fail to create so the precaution added here only

I will remove it if not needed